src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -105,12 +111,11 @@ public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
                 nameDir));
       } else {
         deleteFile(nameDir);
-        nameDir.mkdirs();
```

Comment:
This must have come from IDEA automation in refactoring... still my bad... Good catch!